rialto_airflow/harvest/dimensions.py

```diff
@@ -186,25 +188,28 @@ def fill_in(snapshot: Snapshot, jsonl_file: Path) -> Path:
                 .where(Publication.dim_json.is_(None))
                 .execution_options(yield_per=100)
             )
-            for row in select_session.execute(stmt):
-                try:
-                    dimensions_pub = next(
-                        publications_from_dois([row.doi], batch_size=10)
-                    )
-                except StopIteration:
-                    logging.info(f"No data found for {row.doi}:")
-                    continue
-
-                with get_session(snapshot.database_name).begin() as update_session:
-                    update_stmt = (
-                        update(Publication)  # type: ignore
-                        .where(Publication.doi == row.doi)
-                        .values(dim_json=dimensions_pub)
-                    )
-                    update_session.execute(update_stmt)
-
-                count += 1
-                jsonl_output.write(json.dumps(dimensions_pub) + "\n")
+
```

Comment:
The partitions() method will give us back chunks of results of length equal to the number used in the `yield_per` on the sqlalchemy call.